tests/test_manager.py

```
@@ -920,6 +928,111 @@ def test_split_snapshot_set_prune_bad_source_raises(self):
+        """
+        source_path = "/home"
+
+        # Simulate lvm2-thin (usually higher priority for thin volumes)
+        class Lvm2Thin(MockPlugin):
+            name = "lvm2-thin"
+
+            def __init__(self, logger, plugin_cfg):
+                super().__init__(logger, plugin_cfg)
+                self.priority = 15
+
+            def can_snapshot(self, source):
```

Ruff (0.14.10)

Unused method argument: `source`

(ARG002)